test/parallel/test-net-connect-call-socket-connect.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+// This test checks that calling `net.connect` internally calls
```

Comment:
https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#test-structure

I don't know why it was chosen, but the standard is for the test description to be later in the file.